### ISSUE TITLE
chore: update spin shim to v0.6.0

### ISSUE THREAD

**Issue by Mossaka** [deislabs/containerd-wasm-shims#52]

**Comment by ThorstenHans** [comment]
can I take this one? If so, would it make sense to switch from pinning a `ref` to pinning the desired `tag`?

**Comment by robotastic** [comment]
I am trying to get a small website running using the Spin Shim and Fermyon's Bartholomew project. It looks like they recently updated the .wasm component they include to Spin version 0.6.0: https://github.com/fermyon/bartholomew-site-template/commit/a7624543ca607668405c8a34a4dec8a6d201578e

When I try to run a containerd image that includes that component I get the following error. Is this likely due to incompatible versions of Spin?

```bash
default       7s          Normal    ScalingReplicaSet         deployment/wasm-spin                           Scaled up replica set wasm-spin-5cc95bc6d8 to 1
default       7s          Normal    SuccessfulCreate          replicaset/wasm-spin-5cc95bc6d8                Created pod: wasm-spin-5cc95bc6d8-lffpc
default       6s          Normal    Scheduled                 pod/wasm-spin-5cc95bc6d8-lffpc                 Successfully assigned default/wasm-spin-5cc95bc6d8-lffpc to k3d-wasm-cluster-agent-1
default       6s          Normal    Pulled                    pod/wasm-spin-5cc95bc6d8-lffpc                 Successfully pulled image "mycluster-registry:12345/wasm-fyi:latest" in 41.070916ms
default       6s          Normal    Pulling                   pod/wasm-spin-5cc95bc6d8-lffpc                 Pulling image "mycluster-registry:12345/wasm-fyi:latest"
default       6s          Normal    Pulled                    pod/wasm-spin-5cc95bc6d8-lffpc                 Successfully pulled image "mycluster-registry:12345/wasm-fyi:latest" in 23.109417ms
default       6s          Normal    Created                   pod/wasm-spin-5cc95bc6d8-lffpc                 Created container testwasm
default       6s          Warning   Failed                    pod/wasm-spin-5cc95bc6d8-lffpc                 Error: failed to start containerd task "testwasm": Failed to prepare configuration: unknown
default       4s          Warning   BackOff                   pod/wasm-spin-5cc95bc6d8-lffpc                 Back-off restarting failed container
```

When I run through the example [here](https://github.com/deislabs/containerd-wasm-shims/blob/main/containerd-shim-spin-v1/quickstart.md), everything works correctly.

**Comment by robotastic** [comment]
Nevermind .... I was dumb and messed up my Dockerfile and the files were not copied to the right Directory. Tough to figure that out from the error message!

**Comment by Mossaka** [comment]
> can I take this one? If so, would it make sense to switch from pinning a ref to pinning the desired tag?

sure! please go ahead! Yes, it makes sense to pin the version to the desired tag.

**Comment by Mossaka** [comment]
> Is this likely due to incompatible versions of Spin?

I think that's possible.

**Comment by Mossaka** [comment]
Closed #55
